packages/cpp/ArmoniK.Api.Worker/header/utils/WorkerServer.h

```diff
@@ -41,9 +42,14 @@ class WorkerServer {
    * @brief Constructor for the WorkerServer class.
    * @param configuration A shared pointer to the Configuration object.
    */
-  explicit WorkerServer(const Common::utils::Configuration &configuration) {
-    logger.enrich([](Common::serilog::serilog_context &ctx) { ctx.add("threadId", std::this_thread::get_id()); });
-    logger.add_property("container", "ArmoniK.Worker");
+  explicit WorkerServer(const Common::utils::Configuration &configuration)
+      : logger(Common::logger::writer_console(), Common::logger::formatter_clef()) {
+    logger.local_context_generator_add("threadId", []() {
+      std::stringstream ss;
```

Comment:
Could you be missing #include <sstream> ?

Reply:
As you pointed out offline, it was `#include <thread>`